code/index.html

```diff
@@ -4,27 +4,77 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+<link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="hero-header">
+      <picture>
+        <source srcset="./images/heromsmall.jpg" media="(max-width: 667px)">
+        <source srcset="./images/heromedium.jpg" media="(max-width: 1024px)">
+        <img class="hero-image" src="./images/herobig.jpg" alt="Hero image"> 
+      </picture>
+        <h1 class="tenko">Tenko Museum of Modern Art</h1>
+    </header>
 
-    <!-- Signup form -->
+    <div class="both"> 
+    <section class="faq-section">
+      <h1 class="faq-topic">FAQS</h1>
+    
+    <div class="accordion">
+      <h2 class="header1" id="header1"> 
+        If I want to visit the café, do I need to buy an entrance ticket?</h2>
+        <p class= "content1" id="content1">
+          Yes, you will need to buy an entrance ticket to visit the café.</p>
+    </div>
```

Comment:
Check indentation! And there should be no space between = and value:
`<p class="content1" id="content1">`